geocodeTree.py

```diff
@@ -0,0 +1,25 @@
+import cmritPath as cp
+import osmnx as ox
+import networkx as nx
+import pickle
+
+cf = cp.CmritField()
+
+
+class GeocodeTree:
+    def __init__(self):
+        self.G = cf.graph()
+        self.tree_count = 0
+
+    def add_tree(self, coord):
+        self.tree_count += 1
+        tree_name = f"t-{self.tree_count}"
+        nearest_node = ox.nearest_nodes(self.G, coord[0], coord[1])
+        self.G.add_node(tree_name, x=coord[1], y=coord[0], street_count=3)
+        self.G.add_edge(tree_name, nearest_node)
+        print(self.G.nodes)
```

Comment:
Adds a tree
a variable tree_count is iterated for each click on the "Add Tree" button
trees named in the format t-01, t-02, t-03, ...
adds node with the name

after that the nearest node to the tree is Found. (dependancy scikit-learn added to refactor-code branch)
then add an edge between the tree_node and nearest_node